app/controllers/api/v1/base_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Api::V1::BaseController < ApplicationController
+	rescue_from ActiveRecord::RecordNotFound, with: :not_found!
+
+	def not_found!
+		render json: {error: 'Record Not found'}, status: 404
```

Comment:
Use 2 (not 1) spaces for indentation.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Tab detected.